apscheduler/jobstores/mongodb.py

```diff
@@ -1,14 +1,13 @@
 from __future__ import absolute_import
+
+import pickle
 import warnings
 
-from apscheduler.jobstores.base import BaseJobStore, JobLookupError, ConflictingIdError
-from apscheduler.util import maybe_ref, datetime_to_utc_timestamp, utc_timestamp_to_datetime
 from apscheduler.job import Job
-
-try:
-    import cPickle as pickle
-except ImportError:  # pragma: nocover
-    import pickle
+from apscheduler.jobstores.base import (BaseJobStore, ConflictingIdError,
```

Comment:
Getting rid of the conditional pickle import was fine, but why did you rearrange these other imports? They were fine as they were.

Reply:
I used isort https://github.com/timothycrosley/isort
Shall I unsort?

Reply:
Yes, the sort apparently folds import lines unnecessarily.

Reply:
Unsorted this and the other.

Reply:
There are plenty more imports still in this PR where two import lines were changed into two.

Reply:
Unsorted!